src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ShowPlan/Attributes.cs

```diff
@@ -283,5 +283,32 @@ public bool LongString
         private bool longString = false;
     }
 
+    public sealed class BetterValueAttribute: Attribute
+    {
+        public BetterValueAttribute()
+        {
+            this.value = BetterValue.None;
+        }
+
+        public BetterValueAttribute(BetterValue value){
+            this.value = value;
+        }
+
+        public BetterValue Value
+        {
+            get { return this.value; }
+        }
+
+        private BetterValue value;
```

Comment:
You can get rid of this field and use the `init` keyword to initialize `Value` in the constructor.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/init

Reply:
The property could be rewritten like this: `public BetterValue Value { get; init; }`